package.json

```diff
@@ -16,13 +18,14 @@
   "scripts": {
     "use-prod-db": "cp src/config/firebase.prod.json src/config/firebase.config.json",
     "use-test-db": "cp src/config/firebase.test.json src/config/firebase.config.json",
-    "build": "rm -rf dist && tsc && mkdir -p dist/config && cp src/config/firebase.config.json dist/config/firebase.config.json",
+    "build": "rm -rf dist && tsc && mkdir -p dist/config && cp src/DO_NOT_COMMIT.config.json dist/ && cp src/config/firebase.config.json dist/config/firebase.config.json",
     "start": "node dist/app.js",
     "test": "NODE_ENV=test node dist/app.js",
     "deploy": "yarn build && gcloud app deploy"
   },
   "devDependencies": {
     "@types/node": "^9.6.2",
+    "tslint": "^5.9.1",
```

Comment:
You can add in `prettier` for the autoformatting, as well as including `tslint-prettier` and the `tslint.json` from https://github.com/rahafoundation/raha.io